en/docs/administer/configure-approval.md

```diff
@@ -0,0 +1,21 @@
+# Configure Approval
```

Comment:
Shouldn't this be something along the lines of "Review Subscription Requests"?

Reply:
The UI mentioned here will handle any sort of review request. we've also got environment promotion requests here and will add more later. We could call it "Review Approval Requests"?